src/AirBench/Data/BenchContext.cs

```diff
@@ -15,9 +15,14 @@ public BenchContext() : base("bestBenches")
             Database.SetInitializer(new BenchInitializer());
         }
 
-        void IBenchContext.SaveChanges()
+        int IBenchContext.SaveChanges()
         {
-            base.SaveChanges();
+            return base.SaveChanges();
+        }
+
+        async Task<int> IBenchContext.SaveChangesAsync()
+        {
+            return await base.SaveChangesAsync();
         }
```

Comment:
Did you need to explicitly define these interface methods on this class? You should be able to add methods in the `DbContext` base class to your `IBenchContext` interface without having to define these passthrough methods.